inc/page_settings.php

```diff
@@ -172,6 +172,16 @@ function get_settings_values( $type, $id ){
 	 * @param $post_type
 	 */
 	function add_meta_box( $post_type ){
+		// Don't add meta box to WooCommerce Shop page.
+		if ( class_exists( 'WooCommerce' ) ) {
```

Comment:
It might be slightly neater to just have a single if statement here. It'd look something like this:

```php
$screen = get_current_screen();
if(
    class_exists( 'WooCommerce' ) &&
    ! empty( $screen ) && $screen->id == 'page' &&
    get_option( 'woocommerce_shop_page_id' ) == $_GET['post']
) {
    return;
}
```

Nesting this deep can get slightly messy.

Reply:
Oh. I forgot we're able to do that with wp_option now. 🤦